documentation/meta/project_boards/index.md

```diff
@@ -0,0 +1,25 @@
+# Project boards
+
+Openverse project management consists of three project boards,
+[one for issues](https://github.com/orgs/WordPress/projects/75),
+[one for PRs](https://github.com/orgs/WordPress/projects/98),
+[one for project threads](https://github.com/orgs/WordPress/projects/70) and
+[one for discussions](https://github.com/orgs/WordPress/projects/79).
+
+Issues and PRs are added to these boards and move across columns based on their
+status. Automations to move these cards can be broadly classified into two
+types:
+
+- automations built into the GitHub Projects UI
+- custom automations run as GitHub Actions workflows
+
+Read on to know more about these boards and the automations that apply to them.
+
+```{toctree}
```

Comment:
This is a really cool way to leverage the `toctree` - I hadn't thought about having it at the end of a file like this! (though I'm sure that we have other examples of it in our docs already 😅)